lib/install/validate/validator.go

```diff
@@ -660,6 +660,7 @@ func (v *Validator) reachableRegistries(ctx context.Context, input *data.Data, p
 	if len(input.WhitelistRegistries) > 0 {
 		// ignoring error since default merge policy is union, so should never return
 		// an error
+		// #nosec: Errors unhandled.
 		m, _ := secureRegistriesSet.Merge(insecureRegistriesSet, nil)
```

Comment:
Good point - it would be best to check with @hmahmood, but I think this is good for now. What do you think?